README.md

```diff
@@ -56,13 +56,13 @@ Composer features:
 ## Benefits
 
 <p align="center">
-  <a href="https://storage.googleapis.com/docs.mosaicml.com/images/cost_graph_light.svg#gh-light-mode-only" class="only-light">
-    <img src="https://storage.googleapis.com/docs.mosaicml.com/images/cost_graph_light.svg" width="85%"/>
+  <a href="https://storage.googleapis.com/docs.mosaicml.com/images/time-overview-light.svg#gh-light-mode-only" class="only-light">
+    <img src="https://storage.googleapis.com/docs.mosaicml.com/images/time-overview-light.svg" width="85%"/>
   </a>
   <!-- link to the light mode image even on dark mode, so it will be readable in a new tab -->
   <!-- SETUPTOOLS_LONG_DESCRIPTION_HIDE_BEGIN -->
-  <a href="https://storage.googleapis.com/docs.mosaicml.com/images/cost_graph_light.svg#gh-dark-mode-only" class="only-dark">
-    <img src="https://storage.googleapis.com/docs.mosaicml.com/images/cost_graph_dark.svg" width="85%"/>
+  <a href="https://storage.googleapis.com/docs.mosaicml.com/images/time-overview-dark.svg#gh-dark-mode-only" class="only-dark">
```

Comment:
The link should still go to the light theme, so it will be readable in a new tab.

```suggestion
  <a href="https://storage.googleapis.com/docs.mosaicml.com/images/time-overview-light.svg#gh-dark-mode-only" class="only-dark">
```